src/archivematicaCommon/lib/env_configparser.py

```diff
@@ -70,3 +70,7 @@ def getfloat(self, *args, **kwargs):
     @fallback_option
     def getboolean(self, *args, **kwargs):
         return ConfigParser.SafeConfigParser.getboolean(self, *args, **kwargs)
+
+    @fallback_option
+    def getiboolean(self, *args, **kwargs):
+        return not self.getboolean(*args, **kwargs)
```

Comment:
`tests/test_env_configparser.py` has already good coverage, should we include a test for this new method?

Reply:
I've added new tests for appconfig that test the new "multiple equivalent options" feature.